openpype/hosts/fusion/plugins/create/create_saver.py

```diff
@@ -0,0 +1,190 @@
+import os
+
+import qtawesome
+
+from openpype.hosts.fusion.api import (
+    get_current_comp,
+    comp_lock_and_undo_chunk
+)
+
+from openpype.pipeline import (
+    legacy_io,
+    Creator,
+    CreatedInstance
+)
+
+
+class CreateSaver(Creator):
+    identifier = "io.openpype.creators.fusion.saver"
+    name = "saver"
+    label = "Saver"
+    family = "render"
+    default_variants = ["Main"]
+
+    description = "Fusion Saver to generate image sequence"
+
+    def create(self, subset_name, instance_data, pre_create_data):
+
+        # TODO: Add pre_create attributes to choose file format?
+        file_format = "OpenEXRFormat"
+
+        comp = get_current_comp()
+        with comp_lock_and_undo_chunk(comp):
+            args = (-32768, -32768)  # Magical position numbers
+            saver = comp.AddTool("Saver", *args)
+
+            instance_data["subset"] = subset_name
+            self._update_tool_with_data(saver, data=instance_data)
+
+            saver["OutputFormat"] = file_format
+
+            # Check file format settings are available
+            if saver[file_format] is None:
+                raise RuntimeError(
+                    f"File format is not set to {file_format}, this is a bug"
+                )
+
+            # Set file format attributes
+            saver[file_format]["Depth"] = 0  # Auto | float16 | float32
+            # TODO Is this needed?
+            saver[file_format]["SaveAlpha"] = 1
+
+        self._imprint(saver, instance_data)
+
+        # Register the CreatedInstance
+        instance = CreatedInstance(
+            family=self.family,
+            subset_name=subset_name,
+            data=instance_data,
+            creator=self)
+
+        # Insert the transient data
+        instance.transient_data["tool"] = saver
+
+        self._add_instance_to_context(instance)
+
+        return instance
+
+    def collect_instances(self):
+
+        comp = get_current_comp()
+        tools = comp.GetToolList(False, "Saver").values()
+        for tool in tools:
+
+            data = self.get_managed_tool_data(tool)
+            if not data:
+                data = self._collect_unmanaged_saver(tool)
+
+            # Add instance
+            created_instance = CreatedInstance.from_existing(data, self)
+
+            # Collect transient data
+            created_instance.transient_data["tool"] = tool
+
+            self._add_instance_to_context(created_instance)
+
+    def get_icon(self):
+        return qtawesome.icon("fa.eye", color="white")
+
+    def update_instances(self, update_list):
+        for update in update_list:
+            instance = update.instance
+
+            # Get the new values after the changes by key, ignore old value
+            new_data = {
+                key: new for key, (_old, new) in update.changes.items()
+            }
+
+            tool = instance.transient_data["tool"]
+            self._update_tool_with_data(tool, new_data)
```

Comment:
When the publisher closes I'm getting this error
```
File "C:\Users\jacob\Syncthing\Projekt\OpenPype\openpype\hosts\fusion\plugins\create\create_saver.py", line 95, in update_instances
    key: new for key, (_old, new) in update.changes.items()
AttributeError: 'TrackChangesItem' object has no attribute 'items'
```

No idea why that happens. I'm guessing something have changed in OP from when this part was written.

Reply:
Should be resolved.